source/develop/release-management/features/engine/avoid-ip-spoofing.html.md

```diff
@@ -23,8 +23,8 @@ We should limit traffic that originates from an non-trusted guests to a closed s
 
 ## Current status
 
-*   Link to feature page in a specific release. That release may complete the feature, or parts of it. The complete scope of this feature in this release will be described in the release feature page
-*   Last updated on -- by (WIKI)
+*   Not implemented.
+*   There is workaround with [Network Filter Parameters](http://www.ovirt.org/develop/release-management/features/network/networkfilterparameters/). By setting Network Filter Parameters name as `CTRL_IP_LEARNING` and value as `dhcp` you can reach the same behavior as promised in this feature. (For more safety you can also supply parameter with name `DHCPSERVER` and as value use IP address of any trusted DHCP server.) You can also provide IP addresses manully. By using paramter name `IP` and IP address that is valid on interface as value. You can supply multiple adresses with `IP` (Multiple parameters with name `IP`).
```

Comment:
"You" is uncommon. Please use "the user" as subject or use passive, e.g.
By setting the  vNIC profile's Network Filter to "clean traffic" in the and supplying the relevant Network Filter Parameters on the vNIC, the IP spoofing can be avoided.
For vNICs configured by DHCP the Network Filter Parameter `CTRL_IP_LEARNING` has to be set to `dhcp`. 
To avoid DHCP spoofing, a parameter with name `DHCPSERVER` and the IP address of the trusted DHCP server can also be supplied.